### PR TITLE
Pressure Detection 

### PR DESCRIPTION
**Pull request by CalebKL**
This PR, adds pressure detection feature and increases the Stroke Width when pressure levels changes

### CODE REVIEW

**Review by emmanuelmuturia — COMMENTED** [review]
So far so good...

File: artmaker/src/main/java/io/artmaker/actions/DrawEvent.kt
Comment (emmanuelmuturia) [inline artmaker/src/main/java/io/artmaker/actions/DrawEvent.kt:1]: So far so good. @Fbada006 Why did you use a Float and not a Double or Integer, for example?

Comment (Fbada006) [reply artmaker/src/main/java/io/artmaker/actions/DrawEvent.kt:1]: Because the alpha is supposed to be a float

**Review by Fbada006 — APPROVED** [review]
I think we can ship it like this. We can iterate in future if need be.

I have also added the UI settings for enabling or disabling pressure

**Review by emmanuelmuturia — DISMISSED** [review]

**Review by Fbada006 — APPROVED** [review]

**Pull request merged**